src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/MalformedRoundaboutCheck.java

```diff
@@ -224,7 +225,8 @@ private boolean roundaboutEnclosesRoads(final Route roundabout)
     {
         final Polygon roundaboutPoly = new Polygon(roundabout.asPolyLine());
         return roundabout.start().getAtlas().edgesIntersecting(roundaboutPoly,
-                edge -> HighwayTag.isCarNavigableHighway(edge) && !JunctionTag.isRoundabout(edge)
+                edge -> edge.isMasterEdge() && HighwayTag.isCarNavigableHighway(edge)
```

Comment:
Looks like these changes have been already merged. Could you clean this up please?